tests/functionality/opflow/CMakeLists.txt

```diff
@@ -80,7 +61,7 @@ if(EXAGO_INSTALL_TESTS)
       # This test passes on Ascent...
       set_tests_properties(
         FUNCTIONALITY_TEST_OPFLOW_RAJAHIOP_SPARSE_GPU_TOML_TESTSUITE
-        PROPERTIES LABELS "incline-skip;deception-skip;newell-skip"
+        PROPERTIES LABELS "incline-skip;"
```

Comment:
Reminder to make issue for this - cc @nkoukpaizan in case Crusher/Frontier is having similar issues